Testscripts/Windows/DM-MINMEMHONOR.ps1

```diff
@@ -55,89 +55,97 @@ param([String] $TestParams,
 #
 function Main {
     param (
-        $VM1,
-        $VM2,
-        $TestParams
+        $TestParams, $AllVmData
     )
     $currentTestResult = Create-TestResultObject
     $resultArr = @()
     try {
         $testResult = $null
-        $vm1name = $VM1.RoleName
-        $vm2name = $VM2.RoleName
-        $HvServer = $VM1.HyperVHost
-        $VM1Ipv4 = $VM1.PublicIP
-        $VM2Ipv4 = $VM2.PublicIP
-        $VMPort = $VM2.SSHPort
+        $vm1name = $AllVmData[0].RoleName
```

Comment:
I don't see any benefit to assign complex variable to scalar one here. You just consume another memory

Reply:
I make the ps1 passed parameters consistent.

Reply:
ok, if this helps for code reading, it is fine. But it should be directly used next time.